db/db_impl_secondary.cc

```diff
@@ -52,9 +52,43 @@ Status DBImplSecondary::Recover(
     default_cf_internal_stats_ = default_cf_handle_->cfd()->internal_stats();
     single_column_family_mode_ =
         versions_->GetColumnFamilySet()->NumberOfColumnFamilies() == 1;
+
+    // Recover from all newer log files than the ones named in the
```

Comment:
I spent a little more time thinking about this. We should store the WAL readers as member variables of `DBImplSecondary` because they main still be used for replay after recovery completes.

Reply:
> We should store the WAL readers as member variables of `DBImplSecondary` because they main still be used for replay after recovery completes.
PR updated with log_readers_ as a map from log number to log reader, PTAL